### ISSUE TITLE
Proengsoft\JsValidation\Exceptions\BadRequestHttpException: No validations available for 'email' 

### ISSUE THREAD

**Issue by xoco70** [proengsoft/laravel-jsvalidation#185]
Hi, 

Plugin was working well before, now I got this error, I'm not sure why does it appear...

Here are my rules: 

```
array:3 [▼
  "name" => "required|max:255"
  "email" => "required|email|max:255|unique:users"
  "password" => "confirmed|min:6|required"
]
```

And Here is the full exception

```
Proengsoft\JsValidation\Exceptions\BadRequestHttpException: No validations available for 'email' 
in /Users/julien/Documents/Proyectos/laravel/vendor/proengsoft/laravel-jsvalidation/src/Remote/Validator.php:101
Stack trace:
#0 /Users/julien/Documents/Proyectos/laravel/vendor/proengsoft/laravel-jsvalidation/src/Remote/Validator.php(74): Proengsoft\JsValidation\Remote\Validator->setRemoteValidation('email', Object(Illuminate\Validation\Validator))
#1 /Users/julien/Documents/Proyectos/laravel/vendor/proengsoft/laravel-jsvalidation/src/Remote/Validator.php(43): Proengsoft\JsValidation\Remote\Validator->validateJsRemoteRequest('email')
#2 /Users/julien/Documents/Proyectos/laravel/vendor/proengsoft/laravel-jsvalidation/src/Remote/Resolver.php(84): Proengsoft\JsValidation\Remote\Validator->validate('_jsvalidation', 'email', Array)
#3 [internal function]: Proengsoft\JsValidation\Remote\Resolver->Proengsoft\JsValidation\Remote\{closure}('_jsvalidation', 'email', Array, Object(Illuminate\Validation\Validator))
#4 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(3323): call_user_func_array(Object(Closure), Array)
#5 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(3419): Illuminate\Validation\Validator->callExtension('jsvalidation', Array)
#6 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(557): Illuminate\Validation\Validator->__call('validateJsvalid...', Array)
#7 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(471): Illuminate\Validation\Validator->validateAttribute('_jsvalidation', 'Jsvalidation')
#8 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Validation/ValidatesWhenResolvedTrait.php(21): Illuminate\Validation\Validator->passes()
#9 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Providers/FoundationServiceProvider.php(41): Illuminate\Foundation\Http\FormRequest->validate()
#10 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(1046): Illuminate\Foundation\Providers\FoundationServiceProvider->Illuminate\Foundation\Providers\{closure}(Object(App\Http\Requests\UserRequest), Object(Illuminate\Foundation\Application))
#11 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(1009): Illuminate\Container\Container->fireCallbackArray(Object(App\Http\Requests\UserRequest), Array)
#12 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(663): Illuminate\Container\Container->fireResolvingCallbacks('App\\Http\\Reques...', Object(App\Http\Requests\UserRequest))
#13 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(709): Illuminate\Container\Container->make('App\\Http\\Reques...', Array)
#14 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/RouteDependencyResolverTrait.php(85): Illuminate\Foundation\Application->make('App\\Http\\Reques...')
#15 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/RouteDependencyResolverTrait.php(57): Illuminate\Routing\ControllerDispatcher->transformDependency(Object(ReflectionParameter), Array, Array)
#16 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/RouteDependencyResolverTrait.php(41): Illuminate\Routing\ControllerDispatcher->resolveMethodDependencies(Array, Object(ReflectionMethod))
#17 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(40): Illuminate\Routing\ControllerDispatcher->resolveClassMethodDependencies(Array, Object(App\Http\Controllers\UserController), 'store')
#18 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Route.php(189): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(App\Http\Controllers\UserController), 'store')
#19 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Route.php(144): Illuminate\Routing\Route->runController()
#20 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(642): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#21 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#22 /Users/julien/Documents/Proyectos/laravel/app/Http/Middleware/Authenticate.php(45): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#23 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): App\Http\Middleware\Authenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#24 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/passport/src/Http/Middleware/CreateFreshApiToken.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Laravel\Passport\Http\Middleware\CreateFreshApiToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 /Users/julien/Documents/Proyectos/laravel/app/Http/Middleware/LocaleMiddleware.php(28): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#29 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): App\Http\Middleware\LocaleMiddleware->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(41): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#32 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Illuminate\Routing\Middleware\SubstituteBindings->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(65): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#35 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#38 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#41 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#44 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#47 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#49 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(104): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#50 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(644): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#51 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(618): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#52 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(596): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#53 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(267): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#54 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#55 /Users/julien/Documents/Proyectos/laravel/vendor/proengsoft/laravel-jsvalidation/src/RemoteValidationMiddleware.php(53): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#56 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Proengsoft\JsValidation\RemoteValidationMiddleware->handle(Object(Illuminate\Http\Request), Object(Closure))
#57 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#58 /Users/julien/Documents/Proyectos/laravel/vendor/barryvdh/laravel-debugbar/src/Middleware/Debugbar.php(51): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#59 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Barryvdh\Debugbar\Middleware\Debugbar->handle(Object(Illuminate\Http\Request), Object(Closure))
#60 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#61 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(46): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#62 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(137): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#63 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#64 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(104): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#65 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(149): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#66 /Users/julien/Documents/Proyectos/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(116): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#67 /Users/julien/Documents/Proyectos/laravel/public/index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#68 /Users/julien/.composer/vendor/laravel/valet/server.php(106): require('/Users/julien/D...')
#69 {main}
```

I'm on Laravel 5.3 / PHP 7 / Nginx

Any Idea why is it failing? In screen, it shows me no error, but it blocks POST

**Comment by torrentalle** [comment]
Is your issue resolved in latest version `v1.4.3`?

**Comment by a-komarev** [comment]
Closing due to inactivity. Feel free to reopen it if issue still actual.
